algoliasearch-common/src/main/java/com/algolia/search/inputs/MultipleGetObjectsRequests.java

```diff
@@ -0,0 +1,25 @@
+package com.algolia.search.inputs;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.io.Serializable;
+import javax.annotation.Nonnull;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class MultipleGetObjectsRequests implements Serializable {
```

Comment:
Shouldn't this class be final?

Reply:
No requests classes is final in this client. For consistency purposes I didn't make it final.

Reply:
OKok.